devel/fetch_metrics/fetch_metrics.py

```diff
@@ -83,29 +83,40 @@ def init_args():
     return args
 
 def print_per_verbose(lvl: int, msg: str):
-    if args['verbose'] < lvl: return
+    if args and (args['verbose'] < lvl): return
     tabs = max(0, lvl - 1)
     print("\t" * tabs + msg)
 
 def load_sortings() -> List[Dict[str, Any]]:
     hydrated_sortings = kp.load_json(args['sortingsfile'])
-    return hydrated_sortings
+    assert hydrated_sortings is not None
+    return cast(List[Dict[str, Any]], hydrated_sortings)
 
-def compute_quality_metrics(recording: RecordingExtractor, sorting: SortingExtractor) -> str:
+def compute_quality_metrics(recording: se.RecordingExtractor, sorting: se.SortingExtractor) -> str:
+    # import within function in case we don't have spiketoolkit installed outside the container
+    import spiketoolkit as st
+    expected_version = '0.7.4'
+    assert st.__version__ == expected_version, f'Unexpected spiketoolkit version: {st.__version__} <> {expected_version}'
     return st.validation.compute_quality_metrics(
         sorting, recording,
         metric_names=QUALITY_METRICS, as_dataframe=True).to_dict()
 
-def compare_with_ground_truth(sorting: SortingExtractor, gt_sorting: SortingExtractor):
-    ground_truth_comparison = GroundTruthComparison(gt_sorting, sorting)
+def compare_with_ground_truth(sorting: se.SortingExtractor, gt_sorting: se.SortingExtractor):
+    # import within function in case we don't have spikecomparison installed outside the container
+    import spikecomparison as sc
+    expected_sc_version = '0.3.2'
+    assert sc.__version__ == expected_sc_version, f'Unexpected spiketoolkit version: {sc.__version__} <> {expected_sc_version}'
+    ground_truth_comparison = sc.GroundTruthComparison(gt_sorting, sorting)
 
     return {"best_match_21": ground_truth_comparison.best_match_21.to_list(),
             "best_match_12": ground_truth_comparison.best_match_12.to_list(),
             "agreement_scores": ground_truth_comparison.agreement_scores.to_dict()}
 
 @hi.function(
     'compute_quality_metrics_hi', '0.1.1',
-    kachery_support=True
+    image=spiketoolkit_image,
+    kachery_support=True,
+    modules=['labbox_ephys', 'labbox']
```

Comment:
We need these local modules to be in the container. (Note: if these change then we don't need to rebuild the docker image)